machines/16x8_crossbar/Makefile.machine.include

```diff
@@ -8,17 +8,16 @@ BSG_MACHINE_VCACHE_DMA_DATA_WIDTH     = 32
 BSG_MACHINE_VCACHE_MISS_FIFO_ELS      = 32
 
 BSG_MACHINE_DRAM_SIZE_WORDS           = 536870912  # 2^29 words (2GB)
-BSG_MACHINE_DRAM_BANK_SIZE_WORDS      = 67108864
+BSG_MACHINE_DRAM_BANK_SIZE_WORDS      = 16777216
```

Comment:
Not relevant to this PR, but a question for discussion

Is there a way we can compute this, so that we don't need to change it for every machine?

Reply:
Yeah, we could. That would be less error-prone.

Reply:
OK. Let's "file an issue" to do this in some alternate reality where we have time to do such things.